manifests/init.pp

```diff
@@ -0,0 +1,166 @@
+# Sets up rkhunter to be run once per day
+#
+# @param minute_upd   Cron minute for update check
+# @param hour_upd     Cron hour for update check
+# @param monthday_upd Cron monthday for update check
+# @param month_upd    Cron month for update check
+# @param weekday_upd  Cron weekday for update check
+# @param minute       Cron minute
+# @param hour         Cron hour
+# @param monthday     Cron monthday
+# @param month        Cron month
+# @param weekday      Cron weekday
+#
+# @param package_ensure                The ensure status of packages to be managed
+#
+# @param rkhunter_conf_file            The path to the conf file
+#
+# @param rkhunter_conf_file_template   Template to use for the conf file
+#
+# @param check_for_updates             Check internet for definition updates
+#
+# @author https://github.com/simp/pupmod-simp-rkhunter/graphs/contributors
+#
+class rkhunter (
+  String[1]         $minute_upd                      = '0',
+  String[1]         $hour_upd                        = '0',
+  String[1]         $monthday_upd                    = '*',
+  String[1]         $month_upd                       = '*',
+  String[1]         $weekday_upd                     = '*',
+  String[1]         $minute                          = '10',
+  String[1]         $hour                            = '0',
+  String[1]         $monthday                        = '*',
+  String[1]         $month                           = '*',
+  String[1]         $weekday                         = '*',
+  String[1]         $package_ensure                  = 'latest',
+  String[1]  $rkhunter_conf_file                    = '/etc/rkhunter.conf',
+  String[1]  $rkhunter_conf_file_template           = 'rkhunter/rkhunter-conf.erb',
+  Optional[Boolean]    $check_for_updates            = undef,
+############start of rkhunter.conf options
```

Comment:
> Some of the options with defaults, below, explicitly have instructions to leave them unset so that rkhunter can figure out what the appropriate value is. You may want to re-read the docs to make sure we are not inappropriately setting any of these and then verify they work as desired.
> 
>     ssh_config_dir
>     password_file
>     os_version_file (set via module data)
For these I set them to the corresponding files on our supported operating systems (EL/OEL), so as to not have "magic variables" that the user didn't know what was getting used in these cases. 

That being said, should we just leave those unmanaged then, and let rkhunter do as it will?